### PR TITLE
Added ability to pass flags to RegExp constructor

### PR DESCRIPTION
**Pull request by bmhr**
Added opts.flags to pass to RegExp constructor

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,11 +12,12 @@ module.exports = function (glob, opts) {
   // support single character matching, matching ranges of characters, group
   // matching, etc.
   var extended = opts ? !!opts.extended : false;
-
+
```
Comment (fitzgen) [inline index.js]: Don't add white space, please.

File: index.js [hunk 2]
```diff
@@ -12,11 +12,12 @@ module.exports = function (glob, opts) {
...
   // matching, etc.
   var extended = opts ? !!opts.extended : false;
-
+  
   // If we are doing extended matching, this boolean is true when we are inside
   // a group (eg {*.html,*.js}), and false otherwise.
   var inGroup = false;
-
+  // flags to pass in to RegExp constructor
+  var flags = opts ? String(opts.flags):"";
```
Comment (fitzgen) [inline index.js]: Nitpick: spaces around `:` and full sentences/capitalization/puncuation in comments, please.

**Comment by fitzgen** [comment]
Let me know when you've updated based on review comments and have the Travis-CI integration tests passing.

**Comment by bmhr** [comment]
I have made the changes and It passes the tests.

**Comment by fitzgen** [comment]
Thanks!

**Pull request merged**